middleware.js

```diff
@@ -9,29 +10,17 @@ export const validateAccessToken = async (req, res, next) => {
 };
 
 export const validateJwtToken = async (req, res, next) => {
-  const token = req.headers["id"];
-
+  const token = req.body.token || req.query.token || req.headers["id"];
   if (!token) {
-    res.status(401).send({ message: " token invalid" });
+    res.status(401).send({ message: " please provide user.id" });
```

Comment:
JWT never uses req.headers["id"];. As per assignment, it should come from headers. 
why you taking JWT token from body or query ` req.body.token || req.query.token || req.headers["id"];` ??? @mukeshsin 
Also, your error message is wrong. In case of the wrong token, it should say `Unauthorised!!`.

Reply:
ok i do